cmd/fortifyExecuteScan_test.go

```diff
@@ -0,0 +1,317 @@
+	return &models.IssueFilterSelectorSet{}
+}
+func (f *fortifyMock) GetIssueStatisticsOfProjectVersion(id int64) ([]*models.IssueStatistics, error) {
+	return []*models.IssueStatistics{}, nil
+}
+func (f *fortifyMock) GenerateQGateReport(projectID, projectVersionID, reportTemplateID int64, projectName, projectVersionName, reportFormat string) (*models.SavedReport, error) {
+	return &models.SavedReport{}, nil
+}
+func (f *fortifyMock) GetReportDetails(id int64) (*models.SavedReport, error) {
+	return &models.SavedReport{}, nil
+}
+func (f *fortifyMock) UploadResultFile(endpoint, file string, projectVersionID int64) error {
+	return nil
+}
+func (f *fortifyMock) DownloadReportFile(endpoint string, projectVersionID int64) ([]byte, error) {
+	return []byte{}, nil
+}
+func (f *fortifyMock) DownloadResultFile(endpoint string, projectVersionID int64) ([]byte, error) {
+	return []byte{}, nil
+}
+
+type pullRequestServiceMock struct{}
+
+func (prService pullRequestServiceMock) ListPullRequestsWithCommit(ctx context.Context, owner, repo, sha string, opts *github.PullRequestListOptions) ([]*github.PullRequest, *github.Response, error) {
+	if owner == "A" {
+		result := 17
+		return []*github.PullRequest{&github.PullRequest{Number: &result}}, &github.Response{}, nil
+	} else if owner == "C" {
+		return []*github.PullRequest{}, &github.Response{}, errors.New("Test error")
+	}
+	return []*github.PullRequest{}, &github.Response{}, nil
+}
+
+type execRunnerMock struct {
+	dirValue   string
+	envValue   []string
+	outWriter  io.Writer
+	errWriter  io.Writer
+	executable string
+	parameters []string
+}
+
+func (er *execRunnerMock) SetDir(d string) {
+	er.dirValue = d
+}
+
+func (er *execRunnerMock) SetEnv(e []string) {
+	er.envValue = e
+}
+
+func (er *execRunnerMock) Stdout(out io.Writer) {
+	er.outWriter = out
+}
+
+func (er *execRunnerMock) Stderr(err io.Writer) {
+	er.errWriter = err
+}
+func (er *execRunnerMock) RunExecutable(e string, p ...string) error {
+	er.executable = e
+	er.parameters = p
+	classpathPip := "/usr/lib/python35.zip;/usr/lib/python3.5;/usr/lib/python3.5/plat-x86_64-linux-gnu;/usr/lib/python3.5/lib-dynload;/home/piper/.local/lib/python3.5/site-packages;/usr/local/lib/python3.5/dist-packages;/usr/lib/python3/dist-packages;./lib"
+	classpathMaven := "some.jar\nsomeother.jar"
+	if e == "python2" {
+		er.outWriter.Write([]byte(classpathPip))
+	} else if e == "mvn" {
+		ioutil.WriteFile(strings.ReplaceAll(p[2], "-Dmdep.outputFile=", ""), []byte(classpathMaven), 755)
+	}
+	return nil
+}
+
+func TestVerifyScanResultsFinishedUploading(t *testing.T) {
+	ffMock := fortifyMock{}
+	config := fortifyExecuteScanOptions{DeltaMinutes: 0}
+
+	t.Run("no recent upload detected", func(t *testing.T) {
+		err := verifyScanResultsFinishedUploading(config, &ffMock, 4711, "", &models.FilterSet{}, 0)
+		assert.Error(t, err)
+		assert.Equal(t, "No recent upload detected on Project Version", err.Error())
```

Comment:
You can simplify these two lines:

```suggestion
		assert.EqualError(t, err, "No recent upload detected on Project Version")
```